go/pkg/client/batch_retries_test.go

```diff
@@ -338,7 +338,7 @@ func TestBatchReadBlobsDeadlineExceededRetries(t *testing.T) {
 	client, err := client.NewClient(ctx, instance, client.DialParams{
 		Service:    listener.Addr().String(),
 		NoSecurity: true,
-	}, retrier, client.RPCTimeout(100*time.Millisecond))
+	}, retrier, client.RPCTimeouts(map[string]time.Duration{"default": 100 * time.Millisecond}))
```

Comment:
nit: maybe define this variable at the top of the file and use it here and in the test below.